helm/korifi/values.yaml

```diff
@@ -145,6 +145,8 @@ networking:
   gatewayClass:
 
 experimental:
+  routing:
+    disableRouteController: false
```

Comment:
Could you document the new property under the [experimental section](https://github.com/cloudfoundry/korifi/blob/e43410e8e0762ee3b07e7112c847df1d436b88fd/helm/korifi/values.schema.json#L600-L648) of the values yaml schema? Make sure to also run `make generate` in the root directory to update the [helm values doc](https://github.com/cloudfoundry/korifi/blob/main/README.helm.md)